src/main/java/spoon/reflect/visitor/filter/SameFilter.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * SPDX-License-Identifier: (MIT OR CECILL-C)
+ *
+ * Copyright (C) 2006-2019 INRIA and contributors
+ *
+ * Spoon is available either under the terms of the MIT License (see LICENSE-MIT.txt) of the Cecill-C License (see LICENSE-CECILL-C.txt). You as the user are entitled to choose the terms under which to adopt Spoon.
+ */
+package spoon.reflect.visitor.filter;
+
+import spoon.reflect.declaration.CtElement;
+import spoon.reflect.visitor.Filter;
+
+/** Finds the element given in parameter, useful for checking if an element is in an ancestor */
```

Comment:
maybe add in the doc, that you check with ==. The term "same" is sadly not well known and sometimes used instead of equals.

Reply:
Fixed, thanks!